### ISSUE TITLE
Unused

### ISSUE THREAD

**Issue by Konstantin8105** [elliotchance/c2go#311]
* Function `util.GroupsFromRegex` in according to https://codecov.io/gh/elliotchance/c2go/pull/308/src/util/regexp.go not used
* File `util/test.go` not used in according to https://codecov.io/gh/elliotchance/c2go/pull/308/src/util/test.go

Please clarify - Is it some debug tools?

**Comment by elliotchance** [comment]
Good catch. The easiest way to test if its not used is to rip it out and see if the build still passes ;)

**Comment by Konstantin8105** [comment]
Now, after #344 , it is not so easy.
Some code if not need in according to codeconv, but need for transpiling sqlite.
So, removing unused part will be only after solving all problems.
